dropshot/tests/fail/README.adoc

```diff
@@ -0,0 +1,26 @@
+:showtitle:
+:toc: left
+:icons: font
+
+= Trybuild fail tests
```

Comment:
1. thanks for adding this
2. when this shit often matters is when we update the rust version. i.e. this is when those errors may change and sometimes for the worse. rarely we've filed bugs in rustc; sometimes we've adjusted our tests

Up to you if that's a helpful reminder... either here or closer to the assertion.